base/style/colours.less

```diff
@@ -111,6 +111,29 @@
 .bg-linkedin {background-color:#0077B5;}
 .bg-good-loop {background-color: @gl-red;}
 
+// Social Button
+.btn-facebook {
+	color: #3b5998;
+	background-color: white;
+	border: 1px solid #3b5998;
+
+	&:hover{
+		color: white;
+		background-color: #3b5998;
+	}
+}
+
+.btn-twitter {
+	color: #4099ff;
```

Comment:
FYI, color-twitter and color-facebook are defined earlier in this file -- so you can write e.g. `color: @color-twitter`

It doesn't matter much though, so I'll just merge this in as-is.